Send data to API

khasinski opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/enviromonitor/enviromonitorstation/13)
<!-- Reviewable:end -->

khasinski committed:
Add fake data sending

khasinski committed:
Extract config management to a separate class

khasinski committed:
Move PMS3003 into separate class

khasinski committed:
Add test endpoint data

jiivan commented on EnviroMonitorStation/Config.cpp:
```diff
@@ -0,0 +1,56 @@
+}
+
+void Config::read(const char* fileName) {
+  apiEndpoint = "http://app.smogly.pl/api/v1/station/cef6960f-6627-4c4f-9080-de6bbea41a7e/add-metering/?token=9539f7d1-d633-4838-81cb-6f8c712c6ad0";
```
Is it proper to include token here? Shouldn't it be declared/const independently?

khasinski replied:
It should be, we lack a proper configuration plus format (station id + token) is in flux. I'll extract it once backend will settle on a format.

zen commented:
Review status: 0 of 6 files reviewed at latest revision, 5 unresolved discussions.

---

*[platformio.ini, line 11 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLxqnKJNCiFKjRmJRb:-KfLxqnLpXHUywPrTzVl:b-5w1fc8) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/platformio.ini#L11)):*
> ```INI
> ; http://docs.platformio.org/page/projectconf.html
> 
> ;[env:nodemcuv2]
> ```

Any reason this was removed? We keep it, because in future versions we will probably switch to nodeMCU, bacuse we are running out of PINs on Wemos.

---

*[EnviroMonitorStation/Config.cpp, line 1 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLyXQ8s3lHV51rINrB:-KfLyXQAiNq5msj3Fe7l:bpgw9w9) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/2896261adb8712460472285ac81469c799646c32/EnviroMonitorStation/Config.cpp#L1)):*
> ```cpp
> #include <Arduino.h>
> ```

General note here. To keep this code well organised and compatible with both PIO and Arduino, this library should go into /lib/Config (local library)

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 35 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLzexZBe26HLz0OVL8:-KfLzexZBe26HLz0OVL9:bc9uk4f) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/EnviroMonitorStation/EnviroMonitorStation.ino#L35)):*
> ```Arduino
> #define PMS_SET
> 
> String getMacAddress() {
> ```

Please keep it here, it was intended to be used to generate station_id

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 109 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfMdzwAXN4Vp4GtdU_E:-KfMdzwAXN4Vp4GtdU_F:b-52nbkl) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/2896261adb8712460472285ac81469c799646c32/EnviroMonitorStation/EnviroMonitorStation.ino#L109)):*
> ```Arduino
>     http.POST(output);
>     http.end();
>     delay(4000);
> ```

Please refactor it to millis.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13)*
<!-- Sent from Reviewable.io -->

khasinski commented:
Review status: 0 of 6 files reviewed at latest revision, 5 unresolved discussions.

---

*[platformio.ini, line 11 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLxqnKJNCiFKjRmJRb:-KfMeO7W4GzT7hxeDJtC:b4y4ip7) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/platformio.ini#L11)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

Any reason this was removed? We keep it, because in future versions we will probably switch to nodeMCU, bacuse we are running out of PINs on Wemos.
</blockquote></details>

But no support yet, I usually remove all commented out code :)

---

*[EnviroMonitorStation/Config.cpp, line 1 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLyXQ8s3lHV51rINrB:-KfMeW1j4zfmwT1B_ITL:b6sjm5i) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/2896261adb8712460472285ac81469c799646c32/EnviroMonitorStation/Config.cpp#L1)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

General note here. To keep this code well organised and compatible with both PIO and Arduino, this library should go into /lib/Config (local library)
</blockquote></details>

Will do :+1:

---

*[EnviroMonitorStation/Config.cpp, line 14 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLXxSaX8DhpwmJlRNg-r1-14:-KfMeYEdO16Dv3DL2ide:b3iew3e) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/2896261adb8712460472285ac81469c799646c32/EnviroMonitorStation/Config.cpp#L14)):*
<details><summary><i>Previously, jiivan (Dariusz Rybi) wrote…</i></summary><blockquote>

Is it proper to include token here? Shouldn't it be declared/const independently?
</blockquote></details>

It's not, but it's faster to debug for now. I'll remove it.

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 35 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLzexZBe26HLz0OVL8:-KfMeclgduPF6i5Ed2hj:b-ek6jo5) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/EnviroMonitorStation/EnviroMonitorStation.ino#L35)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

Please keep it here, it was intended to be used to generate station_id
</blockquote></details>

I'll extract a module for that, but where should we use it? Backend doesn't support this, it only uses generated token and station id.

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 109 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfMdzwAXN4Vp4GtdU_E:-KfMelpLb6FFdf42Nik9:b6sjm5i) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/2896261adb8712460472285ac81469c799646c32/EnviroMonitorStation/EnviroMonitorStation.ino#L109)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

Please refactor it to millis.
</blockquote></details>

Will do :+1:

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13)*
<!-- Sent from Reviewable.io -->

zen commented:
Reviewed 1 of 6 files at r1.
Review status: 1 of 6 files reviewed at latest revision, 5 unresolved discussions.

---

*[platformio.ini, line 11 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLxqnKJNCiFKjRmJRb:-KfMfhPG0IHr_6plEyV5:bc5aydg) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/platformio.ini#L11)):*
<details><summary><i>Previously, khasinski (Krzysztof Hasiński) wrote…</i></summary><blockquote>

But no support yet, I usually remove all commented out code :)
</blockquote></details>

The cost is zero and it will let us keep the code ready for new platform.

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 35 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLzexZBe26HLz0OVL8:-KfMfxCrIyZZdEaZsrHi:b-7ed2pf) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/EnviroMonitorStation/EnviroMonitorStation.ino#L35)):*
<details><summary><i>Previously, khasinski (Krzysztof Hasiński) wrote…</i></summary><blockquote>

I'll extract a module for that, but where should we use it? Backend doesn't support this, it only uses generated token and station id.
</blockquote></details>

As discussed, we need to get station_id from unique hardware source. This is the way to do it on ESP8266

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13)*
<!-- Sent from Reviewable.io -->

khasinski committed:
Move helpers to lib folder

khasinski committed:
Move libraries, add DHT22 support

justi committed:
Change DHTPIN + update output info

justi committed:
Update endpoint

khasinski committed:
Add PMS3003 support

justi committed:
Add heat plate controller

khasinski committed:
Adjust format of the endpoint

khasinski committed:
Remove nodemcuv2 support

zen commented:
Review status: 0 of 9 files reviewed at latest revision, 14 unresolved discussions.

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 28 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9hiBWqPE0fO9_ekIK:-Kg9hiBWqPE0fO9_ekIL:b-hccrhj) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L28)):*
> ```Arduino
> #define TIME_BETWEEN_METERINGS 2000
> 
> //# Wemos PIN definitions
> ```

Is it required if PMS is in library?

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 37 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9hqh3KNEoqLdVswQ6:-Kg9hqh3KNEoqLdVswQ7:b-yjwa4k) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L37)):*
> ```Arduino
> #define HEAT_PIN_SWITCH D4
> 
> #define HEAT_PIN_SWITCH D4
> ```

Looks redundant

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 58 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9i2f3axbGl-SNtuJY:-Kg9i2f3axbGl-SNtuJZ:b2dzpfd) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L58)):*
> ```Arduino
> 
>   pinMode(HEAT_PIN_SWITCH, OUTPUT);
>   digitalWrite(HEAT_PIN_SWITCH, HIGH);
> ```

It should not be set to HIGH automatically

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 134 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9ikEFYba9617t0Lr1:-Kg9ikEFYba9617t0Lr2:b-5iyh13) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L134)):*
> ```Arduino
>   root["pm25"] = pm25;
>   root["pm10"] = pm10;
>   root["temp_out1"] = t;
> ```

This is reading from DHT, its actually internal temp and humidity. 

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 135 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9irY933tmYnYTNu-M:-Kg9irY933tmYnYTNu-N:b7w3g0e) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L135)):*
> ```Arduino
>   root["pm10"] = pm10;
>   root["temp_out1"] = t;
>   root["temp_out2"] = 0;
> ```

I'm not sure how to handle missing reading. "0" is really a reading, maybe we should use some crazy number and ignore it in backend

---

*[lib/Config.cpp, line 15 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9j4WQ-5GRT2sZ1tNX:-Kg9j4WQ-5GRT2sZ1tNY:b-y5jkyw) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/lib/Config.cpp#L15)):*
> ```cpp
> void Config::read(const char* fileName) {
>   apiEndpoint = "http://app.smogly.pl/api/v1/station/cef6960f-6627-4c4f-9080-de6bbea41a7e/add-metering/?token=9539f7d1-d633-4838-81cb-6f8c712c6ad0";
> ```

Should not contain token

---

*[lib/DHT22/SmoglyDHT.cpp, line 3 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9jJdtx-oylKm1X9of:-Kg9jJdtx-oylKm1X9og:b-tqyil5) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/lib/DHT22/SmoglyDHT.cpp#L3)):*
> ```cpp
> #include "SmoglyDHT.h"
> #include <DHT.h>
> #define DHTPIN D3     // what digital pin we're connected to
> ```

I think lib should be static, config should be abstracted in EnviroMonitorStation.ino. On the other hand, for out purposes it will be pretty static.

---

*[lib/PMS3003/PMS3003.cpp, line 56 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9je72ohmHmE_31ASe:-Kg9je72ohmHmE_31ASf:b22flta) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/lib/PMS3003/PMS3003.cpp#L56)):*
> ```cpp
> #include "PMS3003.h"
> 
> #define PMS_PIN_SET D5
> ```

I think it will completely overlap what's set in EnviroMonitorStation.ino I'm not sure if it's bad or good ;)

---

*[lib/PMS3003/PMS3003.cpp, line 105 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9kAhcWuF3-MMvRk0C:-Kg9kAhdsGLCJzMI3lF_:bu0m7bv) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/lib/PMS3003/PMS3003.cpp#L105)):*
> ```cpp
>   memset(buf, 0, localPmsDataLen);
> 
>   delay(1000);
> ```

In general, there should be no hidden delays in libraries. Why is this even necessary?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13)*
<!-- Sent from Reviewable.io -->

khasinski committed:
Add param reading from wifi manager

khasinski committed:
Remove unused definitions

khasinski committed:
Disable turning heating on automatically

khasinski committed:
Remove unused file

khasinski commented:
Review status: 0 of 8 files reviewed at latest revision, 14 unresolved discussions, some commit checks failed.

---

*[platformio.ini, line 11 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLxqnKJNCiFKjRmJRb:-KgAJU-LzTI_ClQmWrr9:b2349yz) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/platformio.ini#L11)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

The cost is zero and it will let us keep the code ready for new platform.
</blockquote></details>

There is no guarantee it will work when uncommented.

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 35 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLzexZBe26HLz0OVL8:-KgAJc8hK3q3Wo6uEwk2:b-dbeyqt) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/EnviroMonitorStation/EnviroMonitorStation.ino#L35)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

As discussed, we need to get station_id from unique hardware source. This is the way to do it on ESP8266
</blockquote></details>

We will add it in a separate PR, right now this is just unused code.

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 28 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9hiBWqPE0fO9_ekIK:-KgAJi97zmY-KfMh-9YS:ba3dfob) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L28)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

Is it required if PMS is in library?
</blockquote></details>

Removed

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 37 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9hqh3KNEoqLdVswQ6:-KgAJjn4VsvjNLPZfrWH:ba3dfob) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L37)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

Looks redundant
</blockquote></details>

Removed

---

*[lib/PMS3003/PMS3003.cpp, line 105 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9kAhcWuF3-MMvRk0C:-KgAKSqd6Op-5B4t7VIr:b-39z83v) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/lib/PMS3003/PMS3003.cpp#L105)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

In general, there should be no hidden delays in libraries. Why is this even necessary?
</blockquote></details>

No idea, all examples used that, we can test it without this code.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13)*
<!-- Sent from Reviewable.io -->

zen commented:
Reviewed 7 of 13 files at r2, 1 of 2 files at r3, 2 of 3 files at r4, 1 of 2 files at r6, 2 of 2 files at r7.
Review status: all files reviewed at latest revision, 14 unresolved discussions, some commit checks failed.

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 35 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLzexZBe26HLz0OVL8:-KgALCtLPMlvcfcXUAdB:b32w0x) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/b2a4602a397644e04650ff755838f47643d1a4a6/EnviroMonitorStation/EnviroMonitorStation.ino#L35)):*
<details><summary><i>Previously, khasinski (Krzysztof Hasiński) wrote…</i></summary><blockquote>

We will add it in a separate PR, right now this is just unused code.
</blockquote></details>

OK

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 28 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9hiBWqPE0fO9_ekIK:-KgAJzwHqVSw1D60LeM-:b32w0x) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L28)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

Is it required if PMS is in library?
</blockquote></details>

OK

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 37 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9hqh3KNEoqLdVswQ6:-KgAK18v2PKkfyoHHeb9:b32w0x) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L37)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

Looks redundant
</blockquote></details>

OK

---

*[EnviroMonitorStation/EnviroMonitorStation.ino, line 58 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9i2f3axbGl-SNtuJY:-KgAKA9R7I2DWtup2_Fy:b32w0x) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/EnviroMonitorStation/EnviroMonitorStation.ino#L58)):*
<details><summary><i>Previously, zen (Tomasz Napierala) wrote…</i></summary><blockquote>

It should not be set to HIGH automatically
</blockquote></details>

OK

---

*[lib/Config/Config.cpp, line 1 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLyXQ8s3lHV51rINrB:-KgAKURv338H6-FWU46Z:b32w0x) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/2896261adb8712460472285ac81469c799646c32/lib/Config/Config.cpp#L1)):*
<details><summary><i>Previously, khasinski (Krzysztof Hasiński) wrote…</i></summary><blockquote>

Will do :+1:
</blockquote></details>

OK

---

*[lib/Config/Config.cpp, line 14 at r1](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-KfLXxSaX8DhpwmJlRNg-r1-14:-KgAKnd_ZWcye-HmCrSL:b32wtd) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/2896261adb8712460472285ac81469c799646c32/lib/Config/Config.cpp#L14)):*
<details><summary><i>Previously, khasinski (Krzysztof Hasiński) wrote…</i></summary><blockquote>

It should be, we lack a proper configuration plus format (station id + token) is in flux. I'll extract it once backend will settle on a format.
</blockquote></details>

ok

---

*[lib/PMS3003/PMS3003.cpp, line 105 at r6](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13#-Kg9kAhcWuF3-MMvRk0C:-KgAKwiyJczAZsyhmCue:b-2qbd25) ([raw file](https://github.com/enviromonitor/enviromonitorstation/blob/ce39c4e965f1fc6012539874eaf96b0372f8f1a4/lib/PMS3003/PMS3003.cpp#L105)):*
<details><summary><i>Previously, khasinski (Krzysztof Hasiński) wrote…</i></summary><blockquote>

No idea, all examples used that, we can test it without this code.
</blockquote></details>

Example used it, but in loop(), so it is explicit.
OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/enviromonitor/enviromonitorstation/13)*
<!-- Sent from Reviewable.io -->

khasinski committed:
Use builtin http client, remove unused param
